lib/cisco_node_utils/snmpcommunity.rb

```diff
@@ -22,10 +22,23 @@ class SnmpCommunity < NodeUtil
     @communities = nil
 
     def initialize(name, group, instantiate=true)
-      fail TypeError unless name.is_a?(String) && group.is_a?(String)
+      fail TypeError unless name.is_a?(String)
+      fail TypeError unless group.is_a?(String)
       @name = name
       return unless instantiate
-      config_set('snmp_community', 'community', '', name, group)
+      if platform == :nexus
+        config_set('snmp_community', 'community',
+                   state: '',
+                   name:  @name,
+                   group: group)
+      else
+        config_set('snmp_community', 'community',
+                   state: '',
+                   name:  @name)
+        # create the mapping for group
+        config_set('snmp_community', 'group_simple', state: '', group: group)
```

Comment:
I don't see `state` in the yaml file for the `group_community_mapping` property.